docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/create-edge-syntax.md

```diff
@@ -17,7 +17,7 @@ CREATE EDGE [IF NOT EXISTS] <edge_name>
     | DEFAULT <default_value>
 ```
 
-**Nebula Graph**'s schema is composed of tags and edges, either of which may have properties. `CREATE EDGE` statement defines an edge type with the given name.
+The schema for Nebula Graph is composed of tags and edges, either of which can have properties. The `CREATE EDGE` statement defines an edge type with the given name.
```

Comment:
"The schema for Nebula Graph is composed of tags and edges, and both of them can have properties"?